src/common/ccvi/renderRegionDescription.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { CcviLevel, getCcviLevel } from './index';
+import { formatPercent } from 'common/utils';
+
+export function renderStateDescription(
```

Comment:
should these go in `CCVI_LEVEL_INFO_MAP` 🤔 ? or cleaner to keep them here

Reply:
Maybe moving it to the map would be better so all the level information is in the same place, under a `renderDescription` function property in case we need to pass parameters such as the location name.

Reply:
adding a todo + going to merge in as is for now